book_config.py

```diff
@@ -195,8 +189,13 @@ def search_for_goodreads_url(self):
             self.goodreads_url_edit.setText('')
 
     def update_aliases(self):
+        if 'goodreads.com' not in self.goodreads_url_edit.text():
```

Comment:
yeah i tried it .. just don't want it to fail in the case someone pastes the url in there without the https and or/ www